DirectProgramming/C++SYCL_FPGA/Tutorials/Features/IPAuthoring_interfaces_overview/handshake-control/src/vector_add.cpp

```diff
@@ -0,0 +1,102 @@
+#include <iostream>
+
+// oneAPI headers
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/sycl.hpp>
+
+// Forward declare the kernel name in the global scope. This is an FPGA best
+// practice that reduces name mangling in the optimization reports.
+class SimpleVAdd;
```

Comment:
```suggestion
class IDSimpleVAdd;
```
I won't comment this anymore, but you see the pattern :)